examples/playground/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
```

Comment:
## Todo

- [ ] Make it easy for Panel, awesome-panel and any user to host a Panel `panel-lite-editor` containing some examples.
  - [ ] Style the editor
  - [ ] Put the `panel-lite-editor` in an easy to use js library that can be loaded from a CDN.
  - [ ] Make it possible to provide a dictionary of examples to the `panel-lite-lite` editor
  - [ ] Make it possible to run the `panel-lite-editor` as a Panel `PanelLiteEditor` widget.
  - [ ] Make it possible to toggle the editor part such that you can display the result but users can click some button and editor the code to update the result.